chart/compass/templates/system-fetcher-configmap.yaml

```diff
@@ -0,0 +1,190 @@
+{{if .Values.global.systemFetcher.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: compass-system-fetcher-templates-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+  integration-systems.json: |-
+    [
+      {
+        "id":"{{ (index .Values.global.systemFetcher.intSystems 0).id }}",
```

Comment:
Can we scrap the `global.systemFetcher.intSystems` configuration array altogether and just check in the `upsertIntegrationSystems` function in `loader.go` if an integration system with the given name exists (as we do for the templates)? You could list all integration systems (they should not be many) and iterate by name.

Reply:
It's just that those intSystems ids look tricky to maintain and also why not just simplify things where we can..?

Reply:
A thing to consider is how then to provide the integration system id to those templates that need it in the applicationInputJSON